qgis/examples/elevation_tile_downloader/__init__.py

```diff
@@ -1,10 +1,10 @@
 # -*- coding: utf-8 -*-
 """
 /***************************************************************************
- ElevationTileDownloader
-                                 A QGIS plugin
- Downloads elevation tiles 
- Generated by Plugin Builder: http://g-sherman.github.io/Qgis-Plugin-Builder/
+ElevationTileDownloader
+A QGIS plugin
+Downloads elevation tiles
+Generated by Plugin Builder: http://g-sherman.github.io/Qgis-Plugin-Builder/
```

Comment:
just some linter stuff, apparently I still had the pre-commit hook installed from the last tutorial..